Cargo.toml

```diff
@@ -45,6 +45,7 @@ picky-asn1-x509 = { version = "0.3.2", optional = true }
 users = "0.10.0"
 libc = "0.2.77"
 anyhow = "1.0.32"
+mbox = "0.5.0"
```

Comment:
Maybe put it optional for the `trusted-service` feature?

Reply:
Will do